src/main/java/com/moddy/server/controller/application/ApplicationController.java

```diff
@@ -61,30 +63,30 @@ public SuccessNonDataResponse submitModelApplication(
         hairModelApplicationRegisterService.postApplication(modelId, modelImgUrl, applicationCaptureImgUrl, applicationInfo);
         return SuccessNonDataResponse.success(SuccessCode.CREATE_MODEL_APPLICATION_SUCCESS);
     }
-    @Tag(name = "DesignerController")
+
     @Operation(summary = "[JWT] 디자이너 메인 뷰 조회", description = "디자이너 메인 뷰 조회 API입니다.")
     @ApiResponses({
             @ApiResponse(responseCode = "200", description = "디자이너 메인뷰 조회 성공", content = @Content(schema = @Schema(implementation = DesignerMainResponse.class))),
             @ApiResponse(responseCode = "401", description = "인증 오류 입니다.", content = @Content(schema = @Schema(implementation = ErrorResponse.class))),
             @ApiResponse(responseCode = "500", description = "서버 내부 오류 입니다.", content = @Content(schema = @Schema(implementation = ErrorResponse.class))),
     })
-    @GetMapping(value = "/designer")
+    @GetMapping()
```

Comment:
```suggestion
    @GetMapping
```
p3 소괄호를 넣지 않아도 `/application` 으로 엔드포인트가 적용 됩니다!